torch/nn/modules/module.py

```diff
@@ -1129,7 +1129,8 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                 # This is used to avoid copying uninitialized parameters into
                 # non-lazy modules, since they dont have the hook to do the checks
                 # in such case, it will error when accessing the .shape attribute.
-                is_param_lazy = isinstance(param, torch.nn.parameter.UninitializedParameter)
+                is_param_lazy = isinstance(param, (torch.nn.parameter.UninitializedParameter,
```

Comment:
nit: the lazy package might provide a convenient function to do this to remove this "implementation detail" like `parameter.is_lazy(param)`.

Reply:
Now I implemented `UninitializedMixin` class simplified this line with `isinstance(..., UninitializedMixin)`.